test/e2e/app-dir/app/next.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
   experimental: {
     clientRouterFilterRedirects: true,
+    parallelServerAndEdgeCompiles: true,
```

Comment:
Can we create another test file, override the experimental flags with `webpackBuildWorker` and `parallel` options to test? so they can cover both default build and experimental build.
`app-dir/app` has quite a few test cases, this will loose the default build coverage